console/console-init/oauth-proxy/cfg/oauth-proxy-kubernetes.cfg

```diff
@@ -33,3 +33,5 @@ cookie_secret = "${SSO_COOKIE_SECRET}"
 cookie_domain = "${SSO_COOKIE_DOMAIN}"
 
 skip_provider_button = true
+
+pass_user_headers = true
```

Comment:
With this being default, we can remove the extra argument added if impersonation is enabled: https://github.com/EnMasseProject/enmasse/blob/master/pkg/controller/consoleservice/consoleservice_controller.go#L903

Reply:
Done.